config/detekt.yml

```diff
@@ -22,7 +22,7 @@ formatting:
 complexity:
   TooManyFunctions:
     thresholdInObjects: 14
-    thresholdInClasses: 13
+    thresholdInClasses: 14
```

Comment:
Should we just Suppress this particular issue in the file where it often happens? instead of increasing this parameter each time